config/initializers/assets.rb

```diff
@@ -21,12 +21,16 @@
     modules/ResultsFilter.js
     modules/EducationModule.js
     modules/ResultsModule.js
+    dough/assets/js/lib/*.js
+    dough/assets/js/**/*.js
   )
 
   # Vendor JavaScript
   config.assets.precompile += %w(
     jquery/dist/jquery.js
     jquery-ujs/src/rails.js
+    jqueryThrottleDebounce/jquery.ba-throttle-debounce.js
```

Comment:
TabSelect has a dependency on mediaQueries, which has a dependency on jqueryThrottleDebounce. It wasn't happy until I added it.